File: src/main/java/com/methodpark/cmpe/code/lsp/DecoyDuck.java
package com.methodpark.cmpe.code.lsp;

public class DecoyDuck extends Duck
{
    @Override
    public void display()
    {
        System.out.println("This is a wooden decoy duck.");
    }

    @Override
    public void quack()
    {
        // Override to do nothing.
    }

    @Override
    public void fly()
    {
        // Override to do nothing.
    }

    
}

File: src/main/csharp/Lsp/Lsp/DecoyDuck.cs
using System;

namespace Lsp
{
    public class DecoyDuck : Duck
    {
        public override void Display()
        {
            Console.WriteLine("This is a wooden decoy duck.");
        }

        public override void Quack()
        {
            // Override to do nothing.
        }

        public override void Fly()
        {
            // Override to do nothing.
        }
    }
}